examples/text-to-image/dreamshaper-xl-v2-turbo/pipeline.yaml

```diff
@@ -0,0 +1,21 @@
+runtime:
+  container_commands:
+    - apt-get update
+    - apt-get install -y git
```

Comment:
not sure you strictly need these lines (might speed up the build without them) but up to you